modules/apps/27-interchain-accounts/controller/ibc_middleware.go

```diff
@@ -246,19 +246,6 @@ func (im IBCMiddleware) OnChanUpgradeOpen(ctx sdk.Context, portID, channelID str
 // OnChanUpgradeRestore implements the IBCModule interface
 func (im IBCMiddleware) OnChanUpgradeRestore(ctx sdk.Context, portID, channelID string) {}
 
-// OnChanUpgradeTimeout implements the IBCModule interface
-func (im IBCMiddleware) OnChanUpgradeTimeout(
```

Comment:
we are actually calling `OnChanUpgradeRestore` in the `UpgradeTimeout` application callbacks